proto/prysm/v1alpha1/validator.proto

```diff
@@ -741,6 +765,63 @@ message DutiesResponse {
   }
 }
 
+message DutiesV2Response {
+  option deprecated = true;
+
+  repeated Duty current_epoch_duties = 1;
+
+  repeated Duty next_epoch_duties = 2;
+
+  bytes previous_duty_dependent_root = 3 [ (ethereum.eth.ext.ssz_size) = "32" ];
+  bytes current_duty_dependent_root = 4 [ (ethereum.eth.ext.ssz_size) = "32" ];
+
+  message Duty {
+    // The length of the committee
+    uint64 committee_length = 1;
+
+    // The index into the committee where the validator belongs in.
+    uint64 committee_index = 2
+    [ (ethereum.eth.ext.cast_type) =
+    "github.com/OffchainLabs/prysm/v6/consensus-types/"
+        "primitives.CommitteeIndex" ];
+
+    // The number of committees in the duty's slot.
+    uint64 committees_at_slot = 3;
+
+    // The index of the validator in the committee list
+    uint64 validator_committee_index = 4;
```

Comment:
I am finding this a little bit confusing. What is this and how is it different from `committee_index`? Maybe it is the commentary on `committee_index` which is incorrect.

Reply:
It is the equivalent of the same field by the same name in https://ethereum.github.io/beacon-APIs/#/Validator/getAttesterDuties it's the index of the validator in the committee

Reply:
that's the index of committee in committee(s) it's like a 2d array

Reply:
"The index into the committee where the validator belongs in" is not a proper sentence and I can't understand it

Reply:
Suggestion: "The index of the committee which the given validator has an assignment" or something along those lines. 

Then the `validator_committee_index` should be commented like "The validator's index in the given committee"